tests/test_calc.py

```diff
@@ -125,7 +124,7 @@ def transfer_transaction(
         1.00,  # Expected capital gain/loss
         None,
         {
-            date_to_index(datetime.date(day=1, month=5, year=2020)): {
+            (datetime.date(day=1, month=5, year=2020)): {
```

Comment:
No need in these brackets anymore.
```suggestion
            datetime.date(day=1, month=5, year=2020): {
```

Reply:
I think you missed this one. And there're more places like this below.

Reply:
Yep you're right, updated it now